pkg/venafi/cloud/connector_test.go

```diff
@@ -931,14 +931,14 @@ func TestReadPolicyConfigurationOnlyEC(t *testing.T) {
 		t.Fatalf("%s", err)
 	}
 	expectedPolice := endpoint.Policy{
-		SubjectCNRegexes:         []string{"^[a-z]{1}[a-z0-9.-]*\\.vfidev\\.com$"},
+		SubjectCNRegexes:         []string{"^.*\\.vfidev\\.com$"},
```

Comment:
Did this change on purpose since I update the policy to be less restrictive. This is so e2e test can pass with our regular functions for random string generation we use in our aruba cucumber environment.